README.md

```diff
@@ -1,3 +1,13 @@
 # dockerfiles
 
 This repository contains the Dockerfiles for Objectif Libre's official images.
+
+## CloudKitty
+
+### api
+
+Contains the Dockerfiles for the api side of CloudKitty and its dependencies.
```

Comment:
"the Dockerfile", no S.  "For CloudKitty's API": api side sounds a bit french :wink: